recipe/meta.yaml

```diff
@@ -35,12 +32,25 @@ test:
 about:
   home: http://github.com/pahaz/sshtunnel
   license: MIT
-  license_file: LICENSE
+  license_family: MIT
   summary: 'Pure Python SSH tunnels'
+  description: |
+    sshtunnel Pure python SSH tunnels
+    -------------
+          Author - [Pahaz Blinov](https://github.com/pahaz)
```

Comment:
The Markdown added here isn't parsed somehow in the [README.md](https://github.com/conda-forge/sshtunnel-feedstock/blob/b61f4772d1dcb654d15b0857b738adbf98b90c62/README.md).

Reply:
I removed description from the about section but the Current build status section is still not rendering correctly.

Thoughts?

Reply:
This is probably a `conda-smithy` bug.